dev-cli/container/src/node/src/main.js

```diff
@@ -165,12 +165,13 @@ export const uploadModuleWith =
  */
 export const spawnProcessWith =
   ({ walletExists, readWallet, create }) =>
-    async ({ walletPath, module, tags }) => {
+    async ({ walletPath, module, scheduler, tags }) => {
       if (!(await walletExists(walletPath))) throw new WalletNotFoundError()
 
       const wallet = await readWallet(walletPath)
       const res = await create({
```

Comment:
`create` in `ao-spawn.js` needs to be updated to accept `scheduler` field on the argument.